### PR TITLE
adding missing CVEs from 2013 and 2015.

### PR DESCRIPTION
**Pull request by lcashdol**

### CODE REVIEW

File: 2013/2xxx/CVE-2013-2565.json [hunk 1]
```diff
@@ -1,18 +1,67 @@
 {
```
Comment (cve-team) [inline 2013/2xxx/CVE-2013-2565.json]: CVE id in JSON (2013-2565) does not agree with filename (CVE-2013-2565)!
u'2013-2565' does not match u'^CVE-[0-9]{4}-[0-9]{4,}$'

**Comment by lcashdol** [comment]
Ok, I think I have this fixed.

**Comment by iamleot** [comment]
Hello Larry,
the description fields seems a bit terse (only the software affected
and type of vulnerability seems present).

Can you please provide a bit more context like most other CVE
descriptions?


Thank you!

**Comment by lcashdol** [comment]
Ok, how is that?

**Comment by iamleot** [comment]
Larry W. Cashdollar writes:
> Ok, how is that?

Sorry, IMHO it's still unclear by reading the descriptions where
the vulnerabilities are, how they can be exploted and their possible
effects.

**Comment by lcashdol** [comment]
Ok, I thought these descriptions weren't meant to be very long.  I can flesh out the details.

**Comment by lcashdol** [comment]
Ok, I've added more details to the descriptions and a missing CVE entry.

File: 2013/5xxx/CVE-2013-5654.json [hunk 2]
```diff
@@ -1,18 +1,67 @@
 {
```
Comment (cve-team) [inline 2013/5xxx/CVE-2013-5654.json]: CVE id in JSON (CVE-2013-5655) does not agree with filename (CVE-2013-5654)!
None is not of type u'string'

Comment (lcashdol) [reply 2013/5xxx/CVE-2013-5654.json]: Ok, fixed.

**Comment by iamleot** [comment]
Hello Larry,

"Larry W. Cashdollar" writes:
> Ok, I've added more details to the descriptions and a missing CVE entry. 

Thank you very much!

Only a little possible non-important tiny typo in CVE-2015-4617:

 s/drirectory/directory/


Thank you again!

**Comment by lcashdol** [comment]
Ok. Fixed.

**Comment by cve-team** [comment]
Thank you for your submission, expect CVE-2013-2516, CVE-2013-2565, CVE-2013-5654, CVE-2015-4615, CVE-2015-4617 to be published on http://cve.mitre.org in the next few hours.

**Pull request merged**